.env.development

```diff
@@ -13,4 +13,5 @@ CDN_URL=cdn.plant-for-the-planet.org/development
 # for vegetation change
 SITE_IMAGERY_API_URL=https://raster.plant-for-the-planet.org
 WIDGET_URL=https://widgets.plant-for-the-planet.org
-NEXT_PUBLIC_DONATION_URL=https://donate.plant-for-the-planet.org
\ No newline at end of file
+NEXT_PUBLIC_DONATION_URL=https://donate.plant-for-the-planet.org
+CONFIG_URL=https://app.plant-for-the-planet.org/app/config
```

Comment:
you should use the staging vercel deployment URL over here (for development and staging)

Reply:
@tejassonar config is global for all environments. so you can always use production config.